python/typemap_utils.cpp

```diff
@@ -639,3 +772,65 @@ static int py_list_to_gobj_list(PyObject *po, geometric_object_list *l) {
 
     return 1;
 }
+
+static PyObject *gobj_to_py_obj(geometric_object *gobj) {
+    switch (gobj->which_subclass) {
+    case geometric_object::PRISM: {
+        PyObject *geom_mod = get_geom_mod();
+        PyObject *prism_class = PyObject_GetAttrString(geom_mod, "Prism");
+
+        int num_verts = gobj->subclass.prism_data->vertices.num_items;
+        prism *prsm = gobj->subclass.prism_data;
+
+        PyObject *py_verts = PyList_New(num_verts);
+        for (int i = 0; i < num_verts; ++i) {
+            vector3 vp = prsm->vertices.items[i];
+            vector3 prism_v3 = vector3_plus(prsm->centroid, matrix3x3_vector3_mult(prsm->m_p2c, vp));
```

Comment:
This is correct, but I encapsulated this operation into a subroutine in libctl---you can just say

```
   vector3 prism_v3 = prism_coordinate_p2c(prsm, vp)
```
Here 'c2p' stands for 'cartesian -> prism.' There is also `prism_coordinate_c2p` for converting in the other direction, and `prism_vector_{p2c,c2p}` for converting vectors instead of coordinates (i.e. skip the translation step and just do the rotation).